extra_time_application/tests/test_project_task_save.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (www.elico-corp.com)
+# Elico Proprietary License v1.0.
+# See LICENSE file for full copyright and licensing details.
+from odoo.tests import common
+
+
+class TestProjectTaskSave(common.TransactionCase):
+
+    def setUp(self):
+        super(TestProjectTaskSave, self).setUp()
+        self.name_1 = self.env.ref('project.project_task_9')
+        self.name_2 = self.env.ref('project.project_task_8')
+        self.project_id = self.env.ref('project.project_project_4')
+        self.user_id = self.env.ref('base.user_root')
+        self.sub_extra_time = 0
+        self.timesheet_ids = self.env['account.analytic.line'].write([[
+            0, False, {
+                'date_time': '2017-12-15',
```

Comment:
For all date, I would suggest to use today() and relative dates in the future today()+2 for example